OpenRA.Mods.Common/Widgets/Logic/Editor/MapToolsLogic.cs

```diff
@@ -42,10 +46,11 @@ public MapToolsLogic(Widget widget, World world, ModData modData, WorldRenderer
 
 			var markerToolPanel = widget.Get("MARKER_TOOL_PANEL");
 			toolPanels.Add(MapTool.MarkerTiles, markerToolPanel);
+			var mapGeneratorToolPanel = widget.Get<ScrollPanelWidget>("MAP_GENERATOR_TOOL_PANEL");
```

Comment:
this could also be `widget.GetOrDefault` as to not force mods to define a generator panel

Reply:
Replaced with GetOrNull.